proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java

```diff
@@ -276,7 +277,7 @@ public net.kyori.adventure.text.Component getMotd() {
       if (motd.startsWith("{")) {
```

Comment:
Might it be a good time to ditch JSON support too in this tbh. There are valid JSON text components that don't start with `{` and MiniMessage can represent everything that JSON can so there's no need to still have it.

Reply:
According to what you say, I just checked and any slightly complex JSON formatting in this option was unusable since the [same check](https://github.com/PaperMC/Velocity/blob/dev/3.0.0/proxy/src/main/java/com/velocitypowered/proxy/config/VelocityConfiguration.java#L276) was used to parse JSON 😅

I just added migration support for the JSON format, but I had to add 2 additional `.replace("\\", "")` because otherwise the Gson serializer would add that character, I don't know if it's related to the fix https://github.com/KyoriPowered/adventure/pull/789 but it doesn't work